### PR TITLE
Fsd auto installing dummy

### PR DESCRIPTION
**Pull request by Wyndelle**
Added dummyData for Projects
Recursively installing projects

### CODE REVIEW

File: controller/projects.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ function selectProject({ session, projectAction }) {
             }
           ],
           function (confirm) {
-            if (confirm.install) return res(project);
+            if (confirm.in5stall) return res(project);
```
Comment (jmconnor3) [inline controller/projects.js:77]: did you mean to put a 5 here?

File: test/test-auto-install-projects.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  './github': fakeHelpers,
+  './env': {
+    home: fakeHelpers.home
+  }
+});
+
+const readAndParse = path => JSON.parse(fs.readFileSync(path));
+//recursively grabs projects
+
+describe.only('#installProject()', function () {
```
Comment (jmconnor3) [inline test/test-auto-install-projects.js:23]: remove .only when you push up

File: test/test-test.js [hunk 3]
```diff
@@ -51,20 +51,34 @@ describe('test', function () {
...
-      expect(fs.existsSync(`${path}/test`)).to.be.false;
-      projects.ensureProjectsDirectory();
-      fs.mkdirSync(path);
-      fs.writeFileSync(`${path}/package.json`, '{}');
-      test.grabTests(project).then(function () {
-        expect(fs.existsSync(`${path}/test`)).to.be.true;
-        done();
-      });
-    });
+  // describe('#grabTests()', function () {
```
Comment (jmconnor3) [inline test/test-test.js:54]: why is this commented out?

File: test/test-test.js [hunk 4]
```diff
@@ -51,20 +51,34 @@ describe('test', function () {
...
+          expect(fs.existsSync(`${path}/test`)).to.be.false;
+          projects.ensureProjectsDirectory();
+          fs.mkdirSync(path);
+          fs.writeFileSync(`${path}/package.json`, '{}');
+          test.grabTests(project).then(function () {
+            expect(fs.existsSync(`${path}/test`)).to.be.true;
+            done();
+          });
 
     it('should install package.json if necessary', function (done) {
```
Comment (jmconnor3) [inline test/test-test.js:83]: You should refactor this test so it checks for each project like the test above it.

**Review by jmconnor3 — CHANGES_REQUESTED** [review]

**Review by gregthompson27 — APPROVED** [review]

**Pull request merged**